src/pages/components/search/style.mdx

```diff
@@ -62,12 +62,22 @@ content.
 
 | Class                                             | Property                    | px / rem | Spacing token |
 | ------------------------------------------------- | --------------------------- | -------- | ------------- |
-| `.bx--search--lg`                                 | height                      | 48 / 3   | –             |
-| `.bx--search--sm`                                 | height                      | 32 / 2   | –             |
 | `.bx--search-magnifier` <br/> `.bx--search-close` | height, width               | 16 / 1   | –             |
 | `.bx--search--lg`                                 | padding-left, padding-right | 48 / 3   | `$spacing-09` |
 | `.bx--search--sm`                                 | padding-left, padding-right | 32 / 2   | `$spacing-07` |
 
+## Size
+
+The width of the search field should appropriately fit the design and layout of
+content.
+
+| Size                                              | Height (px / rem)                   | 
```

Comment:
- Add a column before the size column called "Element". The first row under the Element column can say "Input".

Reply:
- Also make sure the heading for the data table size section says "Sizes" instead of "Size"